components/breadcrumb/demo/separator.md

```diff
@@ -1,10 +1,18 @@
 ---
 order: 3
-title: 分隔符
+title: 
+  zh-CN: 分隔符
+  en-US: Separator
 ---
 
+## zh-CN
+
 使用 `separator=">"` 可以自定义分隔符。
 
+## en-US
+
+Use `separator=">"` to custom separator.
```

Comment:
The separator can be customized by setting the `separator` property: `separator=">"`